hack/lib/version.sh

```diff
@@ -161,6 +161,7 @@ kube::version::ldflags() {
       "-X '${KUBE_GO_PACKAGE}/vendor/k8s.io/client-go/pkg/version.${key}=${val}'"
       "-X '${KUBE_GO_PACKAGE}/vendor/k8s.io/kubectl/pkg/version.${key}=${val}'"
```

Comment:
please open a follow up issue to collapse the kubectl pkg/version and consumers of the k8s.io/kubernetes/pkg/version onto component-base/version

Reply:
https://github.com/kubernetes/kubernetes/issues/81731